src/protocol/protocol.yml

```diff
@@ -644,6 +644,7 @@ Page:
         width: number
         height: number
     isClosed: boolean
+    opener: Page?
```

Comment:
Can also remove Page.opener method from the protocol.

Reply:
Done.